files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/svelte_components/index.html

```diff
@@ -144,7 +144,7 @@ <h2 id="Extracting_our_filter_component">Extracting our filter component</h2>
 
 <p>So far so good! Let's try out the app now. You'll notice that when you click on the filter buttons, they are selected and the style updates appropriately. But! We have a problem — the todos aren't filtered. That's because the <code>filter</code> variable flows down from the <code>Todos</code> component to the <code>FilterButton</code> component through the prop, but changes occurring in the <code>FilterButton</code> component don't flow back up to its parent — the data binding is one-way by default. Let's look at a way to solve this.</p>
 
-<h2 id="Sharing_data_between_components_passing_a_handler_as_a_prop">Sharing data between components: passing a handler as a prop</h2>
+<h2 id="Sharing_data_between_components_passFing_a_handler_as_a_prop">Sharing data between components: passing a handler as a prop</h2>
```

Comment:
This extra `F`is clearly a typo
```suggestion
<h2 id="Sharing_data_between_components_passing_a_handler_as_a_prop">Sharing data between components: passing a handler as a prop</h2>
```